---
# Documentation: https://wowchemy.com/docs/managing-content/

title: A comprehensive overview of lncRNA annotation resources
subtitle: ''
summary: ''
authors:
- Jinyuan Xu
- Jing Bai
- Xinxin Zhang
- Yanling Lv
- Yonghui Gong
- Ling Liu
- Hongying Zhao
- Fulong Yu
- Yanyan Ping
- Guanxiong Zhang
- Yujia Lan
- Yun Xiao
- Xia Li
tags: []
categories: []
date: '2017-01-01'
lastmod: 2022-12-08T23:25:18-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-09T04:39:07.314391Z'
publication_types:
- '2'
abstract: Long noncoding RNAs (lncRNAs) are emerging as a class of important regulators
  participating in various biological functions and disease processes. With the widespread
  application of next-generation sequencing technologies, large numbers of lncRNAs
  have been identified, producing plenty of lncRNA annotation resources in different
  contexts. However, at present, we lack a comprehensive overview of these lncRNA
  annotation resources. In this study, we reviewed 24 currently available lncRNA annotation
  resources referring to > 205 000 lncRNAs in over 50 tissues and cell lines. We characterized
  these annotation resources from different aspects, including exon structure, expression,
  histone modification and function.We found many distinct properties among these
  annotation resources. Especially, these resources showed diverse chromatin signatures,
  remarkable tissue and cell type dependence and functional specificity. Our results
  suggested the incompleteness and complementarity of current lncRNA annotations and
  the necessity of integration of multiple resources to comprehensively characterize
  lncRNAs. Finally, we developed 'LNCat' (lncRNA atlas, freely available at http://biocc.hrbmu.edu.cn/LNCat/),
  a user-friendly database that provides a genome browser of lncRNA structures, visualization
  of different resources from multiple angles and download of different combinations
  of lncRNA annotations, and supports rapid exploration, comparison and integration
  of lncRNA annotation resources. Overall, our study provides a comprehensive comparison
  of numerous lncRNA annotations, and can facilitate understanding of lncRNAs in human
  disease.
publication: '*Briefings in Bioinformatics*'
doi: 10.1093/bib/bbw015
---
