---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Systematically prioritizing functional differentially methylated regions (fDMRs)
  by integrating multi-omics data in colorectal cancer
subtitle: ''
summary: ''
authors:
- Huihui Fan
- Hongying Zhao
- Lin Pang
- Ling Liu
- Guanxiong Zhang
- Fulong Yu
- Tingting Liu
- Chaohan Xu
- Yun Xiao
- Xia Li
tags: []
categories: []
date: '2015-01-01'
lastmod: 2022-12-08T23:25:18-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-09T04:39:07.068588Z'
publication_types:
- '2'
abstract: While genome-wide differential DNA methylation regions (DMRs) have been
  extensively identified, the comprehensive prioritization of their functional importance
  is still poorly explored. Here, we aggregated multiple data resources rooted in
  the genome, epigenome and transcriptome to systematically prioritize functional
  DMRs (fDMRs) in colorectal cancer (CRC). As demonstrated, the top-ranked fDMRs from
  all of the data resources showed a strong enrichment for known methylated genes.
  Additionally, we analyzed those top 5% DMR-coupled coding genes using functional
  enrichment, which resulted in significant disease-related biological functions in
  contrast to the tail 5% genes. To further confirm the functional importance of the
  top-ranked fDMRs, we applied chromatin modification alterations of CRC cell lines
  to characterize their functional regulation. Specifically, we extended the utility
  of the top-ranked DMR-coupled genes to serve as classification and survival biomarkers,
  which showed a robust performance across diverse independent data sets. Collectively,
  our results established an integrative framework to prioritize fDMRs, which could
  help characterize aberrant DNA methylation-induced potential mechanisms underlying
  tumorigenesis and uncover epigenome-based biomarkers for clinical diagnosis and
  prognosis.
publication: '*Scientific Reports*'
doi: 10.1038/srep12789
---
