---
# Documentation: https://wowchemy.com/docs/managing-content/

title: 'CUT&RUNTools 2.0: a pipeline for single-cell and bulk-level CUT&RUN and CUT&Tag
  data analysis'
subtitle: ''
summary: ''
authors:
- Fulong Yu
- Vijay G Sankaran
- Guo-Cheng Yuan
tags: []
categories: []
date: '2021-01-01'
lastmod: 2022-12-08T23:21:07-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-09T04:39:07.732223Z'
publication_types:
- '2'
abstract: MOTIVATION Genome-wide profiling of transcription factor binding and chromatin
  states is a widely-used approach for mechanistic understanding of gene regulation.
  Recent technology development has enabled such profiling at single-cell resolution.
  However, an end-to-end computational pipeline for analyzing such data is still lacking.
  RESULTS Here, we have developed a flexible pipeline for analysis and visualization
  of single-cell CUT&Tag and CUT&RUN data, which provides functions for sequence alignment,
  quality control, dimensionality reduction, cell clustering, data aggregation, and
  visualization. Furthermore, it is also seamlessly integrated with the functions
  in original CUT&RUNTools for population-level analyses. As such, this provides a
  valuable toolbox for the community. AVAILABILITY AND IMPLEMENTATION https://github.com/fl-yu/CUT-RUNTools-2.0.
  SUPPLEMENTARY INFORMATION Supplementary data are available at Bioinformatics online.
publication: '*Bioinformatics*'
doi: 10.1093/bioinformatics/btab507
---
