---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Predicting the functions of long noncoding RNAs using RNA-seq based on Bayesian
  network
subtitle: ''
summary: ''
authors:
- Yun Xiao
- Yanling Lv
- Hongying Zhao
- Yonghui Gong
- Jing Hu
- Feng Li
- Jinyuan Xu
- Jing Bai
- Fulong Yu
- Xia Li
tags: []
categories: []
date: '2015-01-01'
lastmod: 2022-12-08T23:25:17-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-09T04:39:07.232313Z'
publication_types:
- '2'
abstract: Long noncoding RNAs (lncRNAs) have been shown to play key roles in various
  biological processes. However, functions of most lncRNAs are poorly characterized.
  Here, we represent a framework to predict functions of lncRNAs through construction
  of a regulatory network between lncRNAs and protein-coding genes. Using RNA-seq
  data, the transcript profiles of lncRNAs and protein-coding genes are constructed.
  Using the Bayesian network method, a regulatory network, which implies dependency
  relations between lncRNAs and protein-coding genes, was built. In combining protein
  interaction network, highly connected coding genes linked by a given lncRNA were
  subsequently used to predict functions of the lncRNA through functional enrichment.
  Application of our method to prostate RNA-seq data showed that 762 lncRNAs in the
  constructed regulatory network were assigned functions. We found that lncRNAs are
  involved in diverse biological processes, such as tissue development or embryo development
  (e.g., nervous system development and mesoderm development). By comparison with
  functions inferred using the neighboring gene-based method and functions determined
  using lncRNA knockdown experiments, our method can provide comparable predicted
  functions of lncRNAs. Overall, our method can be applied to emerging RNA-seq data,
  which will help researchers identify complex relations between lncRNAs and coding
  genes and reveal important functions of lncRNAs.
publication: '*BioMed Research International*'
doi: 10.1155/2015/839590
---
