---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Variant to function mapping at single-cell resolution through network propagation
subtitle: ''
summary: ''
authors:
- Fulong Yu
- Liam D. Cato
- Chen Weng
- L. Alexander Liggett
- Soyoung Jeon
- Keren Xu
- Charleston W. K. Chiang
- Joseph L. Wiemels
- Jonathan S. Weissman
- Adam J. de Smith
- Vijay G. Sankaran
tags: []
categories: []
date: '2022-01-01'
lastmod: 2022-12-08T23:25:18-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-09T04:39:07.151376Z'
publication_types:
- '2'
abstract: Genome-wide association studies in combination with single-cell genomic
  atlases can provide insights into the mechanisms of disease-causal genetic variation.
  However, identification of disease-relevant or trait-relevant cell types, states
  and trajectories is often hampered by sparsity and noise, particularly in the analysis
  of single-cell epigenomic data. To overcome these challenges, we present SCAVENGE,
  a computational algorithm that uses network propagation to map causal variants to
  their relevant cellular context at single-cell resolution. We demonstrate how SCAVENGE
  can help identify key biological mechanisms underlying human genetic variation,
  applying the method to blood traits at distinct stages of human hematopoiesis, to
  monocyte subsets that increase the risk for severe Coronavirus Disease 2019 (COVID-19)
  and to intermediate lymphocyte developmental states that predispose to acute leukemia.
  Our approach not only provides a framework for enabling variant-to-function insights
  at single-cell resolution but also suggests a more general strategy for maximizing
  the inferences that can be made using single-cell genomic data.
publication: '*Nature Biotechnology*'
doi: 10.1038/s41587-022-01341-y
---


this is just a test `how to` use *this* tool